### DISCUSSION TITLE
M2 CVE-2022-24086 and MageCart

### DISCUSSION THREAD

**Discussion by JonLaliberte** [OpenMage/magento-lts#2016] [Questions & Answers] [answered]
I'm wondering if this is a problem for M1 as well?
Details: https://securityaffairs.co/wordpress/127999/hacking/cve-2022-24086-zero-day.html

I also saw this recently: https://sansec.io/research/naturalfreshmall-mass-hack
The root cause of the problem here is an SQL injection in a 3rd party module, but I'm wondering if there is anything that should be patched in `Zend_Memory_Manager`/`Zend_CodeGenerator_Php_File` to prevent this type of vulnerability.

**Comment by Flyingmana** [comment, marked as answer]
we prevented this type of vulnerability by not allowing this kind of unserialization via https://github.com/OpenMage/magento-lts/pull/1251

although it might be worth to check if the sql injection vulnerability could also apply to us

**Reply by JonLaliberte** [reply]
I believe you meant to link to https://github.com/OpenMage/magento-lts/pull/1251?

**Reply by Flyingmana** [reply]
yes, I corrected it, sorry for the confusion, copied to many links recently
